docs/UsersGuide/source/ConfigWorkflow.rst

```diff
@@ -1222,6 +1225,48 @@ Set parameters for customizing the :term:`UPP`.
    
    Note that this variable is first changed to lower case before being used to construct the file names.
 
+RUN_PRDGEN Configuration Parameters
+=====================================
+
+Non-default parameters for the ``run_prdgen`` task are set in the ``task_run_prdgen:`` section of the ``config.yaml`` file.
+
+Basic Task Parameters
+---------------------------------
+For each workflow task, certain parameter values must be passed to the job scheduler (e.g., Slurm), which submits a job for the task.
+
+``TN_RUN_PRDGEN``: (Default: "run_prdgen")   Set the name of this Rocoto workflow task. Users typically do not need to change this value.
+
+``NNODES_RUN_PRDGEN``: (Default: 1) 
+   Number of nodes to use for the job.
+
+``PPN_RUN_PRDGEN``: (Default: 1)
+   Number of :term:`MPI` processes per node.
+
+``WTIME_RUN_PRDGEN``: (Default: 00:30:00)
+   Maximum time for the task to complete.
+
+``MAXTRIES_RUN_PRDGEN``: (Default: 2)
+   Maximum number of times to attempt the task.
+
+``KMP_AFFINITY_RUN_PRDGEN``: (Default: "scatter")
+   Intel Thread Affinity Interface for the ``run_prdgen`` task. See :ref:`this note <thread-affinity>` for more information on thread affinity.
+
+``OMP_NUM_THREADS_RUN_PRDGEN``: (Default: 1) 
+   The number of OpenMP threads to use for parallel regions.
+
+``OMP_STACKSIZE_RUN_PRDGEN``: (Default: "1024m")
+   Controls the size of the stack for threads created by the OpenMP implementation.
+
+``DO_PARALLEL_PRDGEN``: (Default: false)
+   Flag that determines whether to use CFP to run the product generation job in parallel.  This should be used with the RRFS_NA_3km grid and PPN_RUN_PRDGEN should be set to 22.
+
+``ADDNL_OUTPUT_GRIDS``: (Default: [])
+   Set additional output grids for wgrib2 remapping, if any.  Space-separated list of strings, e.g., ( "130" "242" "clue").  Default is no additional grids.
+
+``TESTBED_FIELDS_FN``: (Default: "")
```

Comment:
@BenjaminBlake-NOAA I think config_defaults.yaml had a TESTBED_FIELDS_FN2 field as well (but it looks like that might be gone now?), and both TESTBED_FIELDS_FN* were defined in the MAKE_GRID Config Params section. Is there a reason the second one has been omitted and the first moved to this section?

Reply:
I think what happened was I realized that defining them in the MAKE_GRID Config Params section was a mistake, and since it is only needed for the RUN_PRDGEN task I moved it to there.  Originally there was also a TESTBED_FIELDS_FN2 variable for defining what parameters should go into the testbed files, but I think initially it makes sense to just add one TESTBED_FIELDS_FN variable at this time.